cfme/services/catalogs/orchestration_template.py

```diff
@@ -237,7 +239,7 @@ class AddDialog(CFMENavigateStep):
     VIEW = AddDialogView
 
     def step(self):
-        self.view.configuration.item_select('Create Service Dialog from Orchestration Template')
+        cfg_btn("Create Service Dialog from Orchestration Template")
```

Comment:
@jteehan This is a regression, if the previous call wasn't working we need to debug why, not remove it.

How was the navigation failing before? Can I help debug?